src/test/scala/edu/berkeley/cs/rise/opaque/JoinSuite.scala

```diff
@@ -172,7 +172,7 @@ trait JoinSuite extends OpaqueSQLSuiteBase with SQLHelper {
     }
   }
 
-  ignore("full outer join") {
+  test("full outer join") {
```

Comment:
Can you try adding the full outer join queries in `unsupportedQueries` to `queries` as well please?

Reply:
I think some of those are non-equi joins, so I was going to wait until finishing the BNLJ implementation before adding those.